new-era-commands/slash/help.js

```diff
@@ -18,18 +21,17 @@ module.exports = {
       .addFields(
         {
           name: 'Server Conduct',
-          value:
-            'All conduct must be within our [rules](https://www.theodinproject.com/guides/community/rules), and [community expectations](https://www.theodinproject.com/guides/community/expectations).\nPlease report issues/misconduct to <@575252669443211264>, as well as to ask any rule-related questions. Modmail How-To - <#1059513837197459547>',
+          value: `All conduct must be within our [rules](https://www.theodinproject.com/guides/community/rules), and [community expectations](https://www.theodinproject.com/guides/community/expectations).\nPlease report issues/misconduct to <@${config.modmailUserId}>, as well as to ask any rule-related questions. Modmail How-To - <#${channels.ContactModeratorsChannelId}>`,
```

Comment:
Should we replace the link to the rules page with the #rules channel here? The content should be the same between them, but the rules channel allows users to stay within Discord. On the other hand, it might be weird if the rules is a channel link and community expectations isn't

Reply:
I'm not particularly fussed at the moment with whether it's linked to the website or discord channel, but I do like the consistency of both links next to each other going to the website.

Reply:
Looking at it again, I think keeping them both to the website is the better option; it looks really silly otherwise